Makefile

```diff
@@ -21,14 +21,27 @@ dev staging: assert-dom0 ## Configures and builds a dev or staging environment
 	$(MAKE) prep-dev
 	sdw-admin --apply
 
+.PHONY: dom0-rpm
 dom0-rpm: ## Builds rpm package to be installed on dom0
-	$(MAKE) dom0-rpm-f25
-
-dom0-rpm-f25: ## Builds rpm package to be installed on dom0
-	@./scripts/build-dom0-rpm f25
-
-dom0-rpm-f32: ## Builds rpm package to be installed on dom0
-	@./scripts/build-dom0-rpm f32
+	git clean -fdX rpm-build/
+	@./scripts/build-dom0-rpm
+
+.PHONY: reprotest
+reprotest: ## Runs reprotest for RPMs with all variations
+	reprotest -c "make dom0-rpm" . "rpm-build/RPMS/noarch/*.rpm"
+
+.PHONY: reprotest-ci
+reprotest-ci: ## Runs reprotest for RPMs, with CI-compatible skips in variations
+	# Disable a few variations, to support CircleCI container environments.
+	# Requires a sed hack to reprotest, see .circle/config.yml
+	TERM=xterm-256color reprotest --variations "+all, +kernel, -domain_host, -fileordering" \
+		 -c "make dom0-rpm" . "rpm-build/RPMS/noarch/*.rpm"
+
+.PHONY: install-deps
+install-deps: ## Installs apt package dependencies, for building RPMs
```

Comment:
This may make it more difficult for Fedora-based hosts to build the rpm with this change, since the logic here is Debian-only. Perhaps we can check the distro and conditionally apt or yum install the appropriate dependencies? This used to be enforced by `BuildRequires` in the rpm specfile below

Reply:
According to https://github.com/freedomofpress/securedrop-workstation/issues/598, RPM builds on Fedora _already_ didn't work, so there's no loss of functionality here. If anything, these generalized changes make it a bit more straight forward to run under Fedora, by dynamically switching dependencies as you describe. Whether we even want to support Feodra as an additional build environment is an open question, in my mind, which is why I left #598 as "referenced" only in this PR.